src/components/user/tour/create-tour/tour-detail.js

```diff
@@ -0,0 +1,141 @@
+import React, {Component} from 'react';
+import TextField from "@material-ui/core/TextField";
+import FormControl from '@material-ui/core/FormControl';
+import InputLabel from "@material-ui/core/InputLabel";
+import Input from "@material-ui/core/Input";
+
+export default class TourDetail extends Component {
+
+    constructor(props) {
+        super(props);
+
+    }
+
+    render() {
+        const {currentStepName, handleInputChange, formValues} = this.props;
+
+        return (
+            <div>
+                <h2 className="tour-details-heading">Create new tour - Step 1 - Tour Details</h2>
+                <div className="tour-details-top-block">
+                    <div className="tour-details-left-block">
+                        <h3>Tour Details</h3>
+                        <div className="form-field">
+                            <FormControl>
+                                <InputLabel htmlFor="tourName">Tour Name</InputLabel>
+                                <Input id="tourName"
+                                       name='tourName'
+                                       value={formValues.tourName}
+                                       onChange={(e) => handleInputChange(currentStepName, e)}/>
```

Comment:
never use arrow functions as a callback. We discussed this problem during the classes.